playbook/app/pb_kits/playbook/pb_date_picker/date_picker_helper.ts

```diff
@@ -70,10 +89,10 @@ const datePickerHelper = (config, scrollContainer) => {
     let pluginList = []
 
     // month and week selection
-    if (selectionType === "month" || plugins === true) {
+    if (selectionType === "month" || plugins.length > 0) {
```

Comment:
This part confuses me a little bit, but I think what is going on here is the result of converting to TS which now expects an array _always_. Does that sound right?

Reply:
yes, as plugins is an array, it starts throwing the error `This condition will always return 'false' since the types 'Plugin<{}>[]' and 'boolean' have no overlap.`. The type comes from `flatpickr`, so the way I fixed it was by checking if the array is bigger than 0. Does it make sense?